crates/core/src/factories/core_space.rs

```diff
@@ -214,6 +214,8 @@ struct CoreSpace {
 
 impl Drop for CoreSpace {
     fn drop(&mut self) {
+        tracing::trace!("Dropping core space");
```

Comment:
This was helpful for debugging, I'd like to keep this at `trace` level but I don't have to

Reply:
Seems like tracing to me :+1: